webapp/src/Controller/API/ProblemController.php

```diff
@@ -371,6 +372,40 @@ public function singleAction(Request $request, string $id): Response
         return $this->renderData($request, $object);
     }
 
+    /**
+     * Get the statement for given problem for this contest.
+     * @throws NonUniqueResultException
+     * @Rest\Get("/{id}/statement")
+     * @OA\Response(
+     *     response="200",
+     *     description="Returns the given problem statement for this contest",
+     *     @OA\MediaType(mediaType="application/pdf")
+     * )
+     * @OA\Parameter(ref="#/components/parameters/id")
+     * @OA\Parameter(ref="#/components/parameters/strict")
+     */
+    public function statementAction(Request $request, string $id): Response
+    {
+        $queryBuilder = $this->getQueryBuilder($request)
+            ->setParameter('id', $id)
+            ->andWhere(sprintf('%s = :id', $this->getIdField()));
+
+        $contestProblems = $queryBuilder->getQuery()->getResult();
+
+        if (empty($contestProblems)) {
+            throw new NotFoundHttpException(sprintf('Problem with ID \'%s\' not found', $id));
+        }
+
+        /** @var ContestProblem $contestProblems */
+        $contestProblems = $contestProblems[0][0];
```

Comment:
Why don't we need the handle the case where you get multiple of these problems? We should consider adding a comment to explain why as its currently unclear to me.